packages/shared/examples/range-plot/range-plot.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useMemo } from 'react'
+import { Scale, Scatter } from '@unovis/ts'
+import { VisXYContainer, VisBulletLegend, VisTooltip, VisAxis, VisScatter, VisLine } from '@unovis/react'
+
+import { data, DataRecord, processLineData } from './data'
+
+const height = 1600
+const yScale = Scale.scalePoint([0, 800]).domain(data.map(d => d.occupation))
+const lineData = useMemo(() => processLineData(data), [])
+
+export default function RangePlot (): JSX.Element {
+  const legendItems = [{ name: 'Women', color: '#FF6B7E' }, { name: 'Men', color: '#4D8CFD' }]
+  const style: React.CSSProperties = useMemo(() => ({ width: '100%', height: height }), [])
+  const styleAb: React.CSSProperties = useMemo(() => ({ position: 'absolute', top: 0, left: 0, width: '100%', height: height }), [])
```

Comment:
Correct me if I'm wrong, but I don't think we need this style declaration. The two scatters should render automatically without additional styling.

Reply:
You are absolutely right. Thanks for catching it.